### PR TITLE
Move bulk-scan, bulk-print to .platform.hmcts.net

### PR DESCRIPTION
**Pull request by apereverzin**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
*
*
*

### CODE REVIEW

File: private-dns-config.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ subscriptions:
...
-    zoneTemplate: '${["plum","sscs","idam","camunda","dg","dm","em","et","sptribs","pcq","lau"].contains(product) || ["draft-store-service","div-cos","civil-general-applications", "civil-service", "civil-ccd","civil-ga-ccd","probate-back-office", "rpe-service-auth-provider", "rpe-pdf-service", "sptribs-case-api", "sptribs-frontend", "sptribs-dss-backend"].contains(appFullName) ? environment + ".platform.hmcts.net" : "service.core-compute-" + environment + ".internal"}'
+    zoneTemplate: '${["plum","sscs","idam","camunda","dg","dm","em","et","sptribs","pcq","lau"].contains(product) || ["draft-store-service","div-cos","civil-general-applications", "civil-service", "civil-ccd","civil-ga-ccd","probate-back-office", "rpe-service-auth-provider", "rpe-pdf-service", "sptribs-case-api", "sptribs-frontend", "sptribs-dss-backend", "send-letter-service", "bulk-scan-processor", "bulk-scan-orchestrator", "bulk-scan-payment-processor", "blob-router-service", "reform-scan-notification-service", "bulk-scan-shared-infrastructure", "reform-scan-shared-infra"].contains(appFullName) ? environment + ".platform.hmcts.net" : "service.core-compute-" + environment + ".internal"}'
```
[the remark this answers is not in the document]
Comment (apereverzin) [reply private-dns-config.yml]: @adusumillipraveen Have done according your suggestion.

[the remark this answers is not in the document]
Comment (apereverzin) [reply private-dns-config.yml]: Isn't it what you @adusumillipraveen suggest?

[the remark this answers is not in the document]
Comment (adusumillipraveen) [reply private-dns-config.yml]: added a new suggestion

[the remark this answers is not in the document]
Comment (apereverzin) [reply private-dns-config.yml]: Done @adusumillipraveen

**Review by adusumillipraveen — DISMISSED** [review]

**Comment by apereverzin** [comment]
@adusumillipraveen I have resolved conflicts, please, review

**Pull request closed without merging**